Mergin/plugin.py

```diff
@@ -440,6 +441,16 @@ def remove_remote_project(self):
         except LoginError as e:
             self._login_error_message(e)
 
+    def submit_logs(self):
+        if not self.path:
+            return
+
+        log_file_name, error = send_logs(self.mc.username(), os.path.join(self.path, '.mergin', 'client-log.txt'))
```

Comment:
I would suggest that before sending a log, we show an information message box, saying something like this:
```
This action will send a diagnostic log to the developers.
Use this option when you encounter synchronization issues, as the log is
very useful to determine the exact cause of the problem.
It would be useful if you also send a mail to info@lutraconsulting.co.uk
and briefly describe the problem to add more context to the diagnostic log.

The log does not contain any of your data, only file names. The log file to be sent is located here:
c:\path\to\project\.mergin\client-log.txt

Please click OK if you want to proceed.

[OK]  [Cancel]
```